be/src/vec/exec/join/join_op.h

```diff
@@ -202,6 +206,7 @@ struct RowRefListWithFlag : RowRef {
     friend class ForwardIterator<RowRefListWithFlag>;
 
     Batch<RowRefType>* next = nullptr;
+    ForwardIterator<RowRefListWithFlag> _visited_iter;
```

Comment:
the way will cause unnecessary mem and performance problem

Reply:
fixed